fl4health/clients/scaffold_client.py

```diff
@@ -100,7 +109,9 @@ def update_control_variates(self, local_steps: int) -> None:
         assert self.learning_rate is not None
 
         # y_i
-        client_model_weights = [val.cpu().detach().numpy() for val in self.model.parameters() if val.requires_grad]
+        client_model_weights = [
+            val.cpu().detach().clone().numpy() for val in self.model.parameters() if val.requires_grad
```

Comment:
This is a similar issue to what was happening above with `self.server_model_weights` if we're working exclusively on CPU. I don't think this causes any issues, as it happens after training, but still doing the clone to be safe.